acceptance/Rakefile

```diff
@@ -256,103 +93,69 @@ namespace :ci do
-You may set TESTS=path/to/test,and/more/tests.
-You may set additional Beaker OPTIONS='--more --options'
-If testing from git checkouts, you may optionally set the github fork to checkout from using PUPPET_FORK='some-other-puppet-fork' (you may change the HIERA_FORK and FACTER_FORK as well if you wish).
-You may also optionally set the git server to checkout repos from using GIT_SERVER='some.git.mirror'.
-Or you may set PUPPET_GIT_SERVER='my.host.with.git.daemon', specifically, if you have set up a `git daemon` to pull local commits from.  (You will need to allow the git daemon to serve the repo (see `git help daemon` and the docs/acceptance_tests.md for more details)).
-If there is a Beaker options hash in a ./local_options.rb, it will be included.  Commandline options set through the above environment variables will override settings in this file.
+  $ SHA=<full sha> bundle exec rake ci:test:quick
+
+SHA should be the full SHA for the puppet-agent package.
 EOS
+    task :quick => ['ci:check_env', 'ci:gen_hosts'] do
+      ENV['TESTS'] = get_test_sample.join(",")
+      Rake::Task["ci:test:aio"].invoke
+    end
 
     desc <<-EOS
-Run a limited but representative subset of acceptance tests through Beaker and install packages as part of the AIO puppet-agent installation. This task is intended to reduce testing time on a per-commit basis.
+Run the acceptance tests using puppet-agent (AIO) packages.
 
-  $ env SHA=<full sha> bundle exec rake ci:test:quick
-EOS
-
-  task :quick => ['ci:check_env', 'ci:gen_hosts'] do
-    ENV['TESTS'] = get_test_sample.join(",")
-    beaker_test(:aio)
-  end
+  $ SHA=<full sha> bundle exec rake ci:test:aio
 
-    desc <<-EOS
-Run the acceptance tests through Beaker and install packages as part of the AIO puppet-agent installation.
-#{USAGE}
+SHA should be the full SHA for the puppet-agent package.
 EOS
     task :aio => ['ci:check_env', 'ci:gen_hosts'] do
-      beaker_test(:aio)
+      beaker(:init, '--hosts', ENV['HOSTS'], '--options-file', 'config/aio/options.rb')
+      beaker(:provision)
+      beaker(:exec, 'pre-suite')
+      beaker(:exec, ENV['TESTS'])
+      beaker(:destroy)
     end
 
     desc <<-EOS
-Install puppet as a gem on a predefined set of hosts using Beaker, and run a basic smoke test.
+Run the acceptance tests against puppet gem on various platforms, performing a
+basic smoke test.
 
-  $ env SHA=<full sha> bundle exec rake:ci:gem
+  $ SHA=<full sha> bundle exec rake:ci:gem
+
+SHA should be the full SHA for the component.
 EOS
     task :gem => ['ci:check_env'] do
-      ENV['TESTS'] = 'setup/gem/pre-suite/010_GemInstall.rb'
-
-      # since we specify the hosts file explicitly, we don't need gen_hosts dependency
-      beaker_test(:gem, :hosts => 'config/nodes/gem.yaml')
+      beaker(:init, '--hosts', 'config/nodes/gem.yaml', '--options-file', 'config/gem/options.rb')
+      beaker(:provision)
+      beaker(:exec, 'pre-suite')
+      beaker(:exec, 'setup/gem/pre-suite/010_GemInstall.rb')
```

Comment:
I think it's because I wanted to be able to easily re-run the gem install process. If it was part of the pre-suite, then you'd have to run the whole pre-suite over. That said I agree it's strange having a test that is in the pre-suite directory, but it's not run during the pre-suite group.